Patterns/Chinese/Chinese-DateTime.yaml

```diff
@@ -13,6 +13,8 @@ MonthNumRegex: !simpleRegex
 TwoNumYear: '50'
 YearNumRegex: !simpleRegex
   def: (?<year>((1[5-9]|20)\d{2})|2100)
+IllegalYearRegex: !simpleRegex
```

Comment:
If you want to declare a new regex for this, it should reference IllegalYearRegex, not re-declare it.
And to avoid it being declared everywhere, ideally it should be somewhere like Base-DateTime.yaml